src/common/routing/RouteDefs.jsx

```diff
@@ -25,7 +25,6 @@ const RouteDefs = () => {
         </Route>
         <Route path="/ideas" element={<PlayIdeas />} />
         <Route path="/*" element={<PageNotFound />} />
```

Comment:
This file doesn't have any change except a new line. Can remove that line to mark as unchanged.